templates/redhat/dhcpd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DHCPv4 Server Daemon
```

Comment:
Right, I was thinking this would let you override the ExecStart setting without duplicating the rest, but it only appends settings, causing two ExecStart settings to exist and the unit fails.  Never mind.